config/routes.rb

```diff
@@ -28,6 +28,7 @@
         resource :export, only: [:show], format: 'csv'
       end
     end
+    resources :performances, only: %i[index new create]
```

Comment:
I'd have gone for a [singular resource](https://guides.rubyonrails.org/routing.html#singular-resources) here, as we're only ever working with the "current" percentage, and you get a nicer URL (`/admin/performance`).

That said, this makes it easier to show history if we wanted to do that in the future, so I don't feel strongly about it.